compiler/passes/expandExternArrayCalls.cpp

```diff
@@ -97,6 +100,14 @@ void expandExternArrayCalls() {
 
     if (fcopy) {
       SET_LINENO(fn);
+      Expr* parentScope = fn->defPoint->parentExpr;
+      if (cptrScopes.count(parentScope) == 0) {
+        BlockStmt* useBlock = buildChapelStmt(new UseStmt(new UnresolvedSymExpr("CPtr"), "",
+                                                        true));
```

Comment:
Just so I learn, why don't we just `fn->defPoint->insertAfter(new UseStmt(bla))`?

Reply:
I don't actually know that there's a reason not to.  I arrived at this by pattern matching on other `use` statements that the compiler inserts (specifically, the use of `CPtr` inserted in support of the `extern` block).  But the precise function it called was `static` and fairly general, so I copied out the relevant guts of it.

Reply:
OK thanks, I have no issue with that.